crates/ethcore/src/snapshot/tests/service.rs

```diff
@@ -363,6 +363,8 @@ fn recover_aborted_recovery() {
         }
         e => panic!("Snapshot restoration must be ongoing ; {:?}", e),
     }
+    // abort restoration so that we can delete snapshot root folder
+    service.abort_restore();
```

Comment:
why call it twice?

Reply:
Poor attempt to test something. Will remove it.